### PR TITLE
Feedback PR

### PR DESCRIPTION
**Pull request by crlnvls**

### CODE REVIEW

**Comment by tsmjohnpascual** [comment]
Guys this is insane! Well done!

**Review by MoGoni52 — COMMENTED** [review]
Website looks and works really well! Functionality and design is really great - very similiar to Github so excellent work. I really like how you also deployed to Netlify aswell. I have read through the code and everything looks good and it is easy to follow. I have noticed you did not use redux in your application - any reason why?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Github Repo Tracker
```
Comment (tsmjohnpascual) [inline README.md:1]: Great read me!

File: src/App.jsx [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
+              <Pages.User />
+            </>
+          }
+        ></Route>
+        <Route
+          path="*"
+          element={
+            <>
+              <Navbar />
+              <Pages.NothingFound />
```
Comment (tsmjohnpascual) [inline src/App.jsx:35]: Nice addition

File: src/components/Card/index.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import moment from "moment";
```
Comment (tsmjohnpascual) [inline src/components/Card/index.jsx:3]: what is a moment?

File: src/pages/User/index.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      console.log(error)
+    }
+  };
+
+  return (
+    <>
+      <SearchForm getUser={getUser} getRepo={getRepo} />
+      <div className="result" style={{ display: !user ? "none" : "flex" }}>
+        <div className="user-info">
+          <img src={user && user.data["avatar_url"]} alt="image" />
```
Comment (tsmjohnpascual) [inline src/pages/User/index.jsx:49]: what's with the '&&' in this line?

File: src/App.jsx [hunk 5]
```diff
@@ -1,34 +1,44 @@
...
+            </>
+          }
+        ></Route>
+      </Routes>
```
Comment (MoGoni52) [inline src/App.jsx:39]: Nice clear routes defined.

**Pull request closed without merging**